.github/workflows/pullrequest-trusted.yml

```diff
@@ -8,24 +8,23 @@ concurrency:
 on:
   pull_request_target:
     branches: [ 'main' ]
-    types: [ 'labeled' ]
 
 # To test workflow updates you need to work in a branch directly on viamrobotics/viam-cartographer
 # and tag your working branch instead of @main in any viamrobotics/viam-cartographer "uses" below.
 # Don't forget to tag back to @main before merge.
 
 jobs:
   test:
-    if: github.event.label.name == 'safe to test' || 
```

Comment:
note -- you probably copied this complex `@main` stuff from a repo that accepts frequent outside contributions

as long as this repo keeps the 'Require approval for all outside collaborators' setting, you can change `viamrobotics/viam-cartographer/.github/workflows/test.yml@main` calls to `./.github/workflows/test.yml`, which makes life easier in a bunch of cases

(but up to you whether to make this change)

Reply:
so with this change i won't have to jump thru hoops to test a change to the test workflow?

Reply:
yup exactly